shared/js/ui/models/site.es6.js

```diff
@@ -185,7 +185,15 @@ Site.prototype = window.$.extend({},
     getUniqueTrackersBlockedCount: function () {
       // console.log('[model] getUniqueTrackersBlockedCount()')
       const count = Object.keys(this.tab.trackersBlocked).reduce((total, name) => {
-        return this.tab.trackersBlocked[name].urls.length + total
```

Comment:
Had to update getUniqueTrackersBlockedCount() and getUnknownTrackersCount() because of the new trackers structure. In both function there was a direct length check on the urls array, which is now an object.
Also added a bit more defensive checks just to make sure we don't error on Object.keys() and the like